docs/core/porting/snippets/upgrade-assistant-wpf-framework/csharp/WebSiteRatings/Models/Site.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Data.Sqlite;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WebSiteRatings.Models
+{
+    public class Site
+    {
+        public string Title;
+        public int Rating;
+        public string Url;
+        public long Id = -1;
+
+        public Site(string title, int rating, string url, long id = -1)
+        {
+            Title = title;
+            Rating = rating;
+            Url = url;
+            Id = id;
+        }
+
+        public void Save()
+        {
+            using (SqliteConnection connection = Database.OpenConnection())
+            {
+                connection.Open();
+
+                using (SqliteCommand saveCmd = connection.CreateCommand())
+                {
+                    if (Id == -1)
+                    {
+                        saveCmd.CommandText = $"INSERT INTO Sites (Title, Rating, URL) VALUES ('{Title}', {Rating}, '{Url}');";
```

Comment:
We should pass `Title`, `Rating`, and `Url` in as parameters rather than directly in the command text. Same in the update command below and in the delete.